### ISSUE TITLE
Missing Texture for Tall Grass and Tulips

### ISSUE THREAD

**Issue by howyay** [JosefLitos/Enhanced-Faithcraft#2]
Hi! Thank you for this wonderful resource pack, it really deserves more attention and love <3

As seen in the screenshot below, currently (1.7.2 downloaded from Modrinth on 1.20.4), tall grass and tulips would show the missing texture. The tulips shows as cubes of the missing-texture texture, not portrayed in the screenshot. I'm not very familiar with the source yet, but I assumed this is due to some placeholders and a known issue. (?)

![image](https://github.com/JosefLitos/Enhanced-Faithcraft/assets/28365438/c08eb216-2b0d-468e-ac1f-27968b51835a)

On another note, are you open to PRs? Is there tracking for the completeness of this pack? I really enjoy it, and I would love to help in any shape or form.

**Comment by litoj** [comment]
Hi, this seems really odd. Could you post the logs? It should contain some error message as to what it is missing. I know the short grass naming changed, which is the cause for the tall grass lower half, but I am not aware of a tulip change.

I think this isn't a difficut fix, but I will have to split the releases for 1.20.4+ because of their renaming, so a PR would unfortunately solve only a part of the issue.

**Comment by litoj** [comment]
The fix is to replace `block/grass` with `block/short_grass` in `assets/minecraft/models/block/tall_grass_bottom.json` and in all `assets/minecraft/models/block/3d/flower/*_tulip.json` replace `block/tulip/orange` with `block/3d/flower/orange_tulip`

the short grass model isn't applied at all also because of the rename, I want to deal with this in the next release, so these fixes might take a little extra time.

**Comment by litoj** [comment]
Also, are you sure you're on 1.7.2? The tulips broke because of a change I made in 1.7.3.

Sidenote: I am removing the 1.7.3 release, because it doesn't follow semver. I removed a lot of varied textures in there and replaced some with model variation, I will try to keep the old style going without the removal of extra textures alongside my intended direction for people that want more variation and use x32 packs anyway.

**Comment by howyay** [comment]
You are right, I was downloading with Prism launcher, and was convinced that I had downloaded 1.7.2. I then tried 1.7.2 directly from Modrinth website, which did not have this issue with either tall grass or tulips, so I must've been mistaken - sorry for the confusion. 

Regardless, here's the relevant log entry on 1.7.3:
```
[17:49:11] [CullThread/WARN]: Unable to load model: 'minecraft:block/tulip/orange' referenced from: minecraft:white_tulip#: java.io.FileNotFoundException: minecraft:models/block/tulip/orange.json
```
Aligns well with your proposed solution from earlier. 

I also should've noted earlier that orange tulips are fine, its only the non-orange ones that have the missing-texture texture.

Thank you again for your time.
